bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/TimingWindowDialog.java

```diff
@@ -1,381 +0,0 @@
-package jsky.app.ot.gemini.editor;
```

Comment:
Moved to the new package and heavily modified (although the appearance is the same).